backend/src/api/player_view_functions.clj

```diff
@@ -8,13 +8,19 @@
   [game-state player-id]
   (get-in game-state [:players (conversions/player-num game-state player-id) :hand]))
 
+(defn get-row-cards
+  "Returns the cards as seen by the player"
+  [row-cards game-state player-id]
+  (mapv (fn [card]
+          (assoc card :owner
+                 (conversions/translate-player game-state (:owner card) player-id)))
+        row-cards))
+
 (defn get-rows
   "Return the rows as seen by the player"
   [game-state player-id]
-  (mapv #(mapv (fn [card]
-                 (assoc card :owner
-                   (conversions/translate-player game-state (:owner card) player-id)))
-               %)
+  (mapv (fn [row]
+          (get-row-cards (:cards row) game-state player-id))
```

Comment:
You are currently eliminating in this step any information other than the contents of `:cards` from a row. Is this what you intend?

Reply:
It is **right now**.
My only intent with this pull-request is to change the backend so it handles row limits, but frontend will be done in another pull-request (for #90).
So the api should keep giving the same as before to frontend. That's why here it loses information.

Reply:
I thought so, but wanted to double-check and comment so we don't forget 🙃